ThunderTable/InputDatePickerViewCell.swift

```diff
@@ -8,13 +8,17 @@
 
 import UIKit
 
-open class InputDatePickerViewCell: TableViewCell {
+open class InputDatePickerViewCell: TableViewCell, DatePickerCell {    
```

Comment:
Could we doc these subclasses while we are here to say what they do, would be easier to understand :)